### ISSUE TITLE
Ping test failing under Windows

### ISSUE THREAD

**Issue by guilhemmarchand** [seunomosowon/TA-connectivity#6]
Hello,

Thank you for your work, however it looks like there is an issue while running under Windows.
Ping tests systematically fail with error code "997", while the same setup works perfectly with a Linux based OS box.

Any chance you get a fix ?

Thank you

**Comment by seunomosowon** [comment]
Yeah. I have made some updates, but working on testing this summer.
Currently an error with the response for Windows.

I’ll push out an update in 2 weeks.

**Comment by guilhemmarchand** [comment]
Thank you very much for you quick reply !
Anything you could share before even in a testing state ?
I am working just this week for the customer (I’m a Splunk PS) and you app does perfectly what we need ;-)

**Comment by seunomosowon** [comment]
I'll send you something at the end of day
